{
  "uid" : "a0994c9c10b396b7",
  "name" : "User can transfer funds between accounts",
  "fullName" : "tests.core_features.test_funds_transfer.TestFundsTransfer#test_successful_funds_transfer",
  "historyId" : "9de203c446da32242737303068260b58",
  "time" : {
    "start" : 1705842737280,
    "stop" : 1705842739203,
    "duration" : 1923
  },
  "status" : "skipped",
  "statusMessage" : "XFAIL reason: Execution blocked by Cloudflare\n\n_pytest.outcomes.XFailed: Execution blocked by Cloudflare",
  "statusTrace" : "tests/core_features/test_funds_transfer.py:26: in test_successful_funds_transfer\n    block_checker(driver)\n        driver     = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"09d6cdf164d75afb16565cbec08fc92a\")>\n        new_user   = NewUser(first_name='Russell', last_name='Reid', address='9068 Scott Creek Suite 412\\nWest Diana, MH 06260', city='Davi...='+1-516-335-2592x943', ssn='228-92-5861', username='uedwards', password='!OwHUfILo5', email='smithtaylor@example.org')\n        register_page = <src.pages.register_page.RegisterPage object at 0x7f5da60f5510>\n        self       = <test_funds_transfer.TestFundsTransfer object at 0x7f5da92983d0>\nsrc/utils/block_checker.py:8: in block_checker\n    pytest.xfail(reason=\"Execution blocked by Cloudflare\")\nE   _pytest.outcomes.XFailed: Execution blocked by Cloudflare\n        driver     = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"09d6cdf164d75afb16565cbec08fc92a\")>",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "_session_faker",
    "time" : {
      "start" : 1705842674470,
      "stop" : 1705842674491,
      "duration" : 21
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "stepsCount" : 0,
    "hasContent" : false
  }, {
    "name" : "Health check request",
    "time" : {
      "start" : 1705842674491,
      "stop" : 1705842674944,
      "duration" : 453
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "stepsCount" : 0,
    "hasContent" : false
  }, {
    "name" : "Initialize driver",
    "time" : {
      "start" : 1705842736443,
      "stop" : 1705842737279,
      "duration" : 836
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "stepsCount" : 0,
    "hasContent" : false
  } ],
  "testStage" : {
    "status" : "skipped",
    "statusMessage" : "XFAIL reason: Execution blocked by Cloudflare\n\n_pytest.outcomes.XFailed: Execution blocked by Cloudflare",
    "statusTrace" : "tests/core_features/test_funds_transfer.py:26: in test_successful_funds_transfer\n    block_checker(driver)\n        driver     = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"09d6cdf164d75afb16565cbec08fc92a\")>\n        new_user   = NewUser(first_name='Russell', last_name='Reid', address='9068 Scott Creek Suite 412\\nWest Diana, MH 06260', city='Davi...='+1-516-335-2592x943', ssn='228-92-5861', username='uedwards', password='!OwHUfILo5', email='smithtaylor@example.org')\n        register_page = <src.pages.register_page.RegisterPage object at 0x7f5da60f5510>\n        self       = <test_funds_transfer.TestFundsTransfer object at 0x7f5da92983d0>\nsrc/utils/block_checker.py:8: in block_checker\n    pytest.xfail(reason=\"Execution blocked by Cloudflare\")\nE   _pytest.outcomes.XFailed: Execution blocked by Cloudflare\n        driver     = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"09d6cdf164d75afb16565cbec08fc92a\")>",
    "steps" : [ {
      "name" : "Navigate to Register page",
      "time" : {
        "start" : 1705842737280,
        "stop" : 1705842738019,
        "duration" : 739
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "stepsCount" : 0,
      "hasContent" : false
    }, {
      "name" : "Register a new user",
      "time" : {
        "start" : 1705842738019,
        "stop" : 1705842739202,
        "duration" : 1183
      },
      "status" : "failed",
      "statusMessage" : "_pytest.outcomes.XFailed: Execution blocked by Cloudflare\n",
      "statusTrace" : "  File \"/usr/workspace/tests/core_features/test_funds_transfer.py\", line 26, in test_successful_funds_transfer\n    block_checker(driver)\n  File \"/usr/workspace/src/utils/block_checker.py\", line 8, in block_checker\n    pytest.xfail(reason=\"Execution blocked by Cloudflare\")\n  File \"/usr/local/lib/python3.11/site-packages/_pytest/outcomes.py\", line 256, in xfail\n    raise XFailed(reason)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "stepsCount" : 0,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : true,
    "stepsCount" : 2,
    "hasContent" : true
  },
  "afterStages" : [ {
    "name" : "Initialize driver::0",
    "time" : {
      "start" : 1705842739210,
      "stop" : 1705842739266,
      "duration" : 56
    },
    "status" : "passed",
    "steps" : [ {
      "name" : "Terminate driver",
      "time" : {
        "start" : 1705842739210,
        "stop" : 1705842739266,
        "duration" : 56
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "stepsCount" : 0,
      "hasContent" : false
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "stepsCount" : 1,
    "hasContent" : true
  } ],
  "labels" : [ {
    "name" : "feature",
    "value" : "PARA-15270: Funds transfer"
  }, {
    "name" : "epic",
    "value" : "PARA-15242: Funds operations"
  }, {
    "name" : "severity",
    "value" : "Critical"
  }, {
    "name" : "tag",
    "value" : "smoke"
  }, {
    "name" : "parentSuite",
    "value" : "tests.core_features"
  }, {
    "name" : "suite",
    "value" : "test_funds_transfer"
  }, {
    "name" : "subSuite",
    "value" : "TestFundsTransfer"
  }, {
    "name" : "host",
    "value" : "63e198265aea"
  }, {
    "name" : "thread",
    "value" : "1-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.core_features.test_funds_transfer"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "critical",
    "retries" : [ {
      "uid" : "ce1820022039b881",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: stale element not found\n  (Session info: headless chrome=119.0.6045.159); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception",
      "time" : {
        "start" : 1705842734259,
        "stop" : 1705842736358,
        "duration" : 2099
      }
    }, {
      "uid" : "76d23de3809cf139",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: stale element not found\n  (Session info: headless chrome=119.0.6045.159); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception",
      "time" : {
        "start" : 1705842731302,
        "stop" : 1705842733335,
        "duration" : 2033
      }
    }, {
      "uid" : "a745ad5ee6f3759e",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: stale element not found\n  (Session info: headless chrome=119.0.6045.159); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception",
      "time" : {
        "start" : 1705842728185,
        "stop" : 1705842730342,
        "duration" : 2157
      }
    } ],
    "categories" : [ ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 0,
        "skipped" : 5,
        "passed" : 10,
        "unknown" : 0,
        "total" : 15
      },
      "items" : [ {
        "uid" : "aa8ed74aaffaf2c3",
        "status" : "passed",
        "time" : {
          "start" : 1705837885137,
          "stop" : 1705837891160,
          "duration" : 6023
        }
      }, {
        "uid" : "6c8a70f7cc2ef5a0",
        "status" : "passed",
        "time" : {
          "start" : 1705747053304,
          "stop" : 1705747060297,
          "duration" : 6993
        }
      }, {
        "uid" : "61ffb11def93c27c",
        "status" : "passed",
        "time" : {
          "start" : 1705596699352,
          "stop" : 1705596706229,
          "duration" : 6877
        }
      }, {
        "uid" : "cf616df103281f78",
        "status" : "passed",
        "time" : {
          "start" : 1705440534129,
          "stop" : 1705440542856,
          "duration" : 8727
        }
      }, {
        "uid" : "eee6fe3280500034",
        "status" : "passed",
        "time" : {
          "start" : 1705392736551,
          "stop" : 1705392743691,
          "duration" : 7140
        }
      }, {
        "uid" : "726d974e0ff66fb5",
        "status" : "passed",
        "time" : {
          "start" : 1705147582134,
          "stop" : 1705147589447,
          "duration" : 7313
        }
      }, {
        "uid" : "94de6cc23fcea23d",
        "status" : "passed",
        "time" : {
          "start" : 1705016467742,
          "stop" : 1705016474637,
          "duration" : 6895
        }
      }, {
        "uid" : "a01fb15f1a324b71",
        "status" : "passed",
        "time" : {
          "start" : 1704912992925,
          "stop" : 1704912999078,
          "duration" : 6153
        }
      }, {
        "uid" : "e26db93d7d33b0dd",
        "status" : "skipped",
        "statusDetails" : "XFAIL reason: Health check failed with status code 502\n\n_pytest.outcomes.XFailed: Health check failed with status code 502",
        "time" : {
          "start" : 1704908682716,
          "stop" : 1704908682716,
          "duration" : 0
        }
      }, {
        "uid" : "994a0b144ed3673c",
        "status" : "passed",
        "time" : {
          "start" : 1704874953971,
          "stop" : 1704874960869,
          "duration" : 6898
        }
      }, {
        "uid" : "2962ff7f92f9874",
        "status" : "skipped",
        "statusDetails" : "XFAIL reason: Execution blocked by Cloudflare\n\n_pytest.outcomes.XFailed: Execution blocked by Cloudflare",
        "time" : {
          "start" : 1704839516829,
          "stop" : 1704839518594,
          "duration" : 1765
        }
      }, {
        "uid" : "979f1a264a135410",
        "status" : "skipped",
        "statusDetails" : "XFAIL reason: Execution blocked by Cloudflare\n\n_pytest.outcomes.XFailed: Execution blocked by Cloudflare",
        "time" : {
          "start" : 1704789324653,
          "stop" : 1704789326327,
          "duration" : 1674
        }
      }, {
        "uid" : "aa45d9569d232db2",
        "status" : "passed",
        "time" : {
          "start" : 1704759014192,
          "stop" : 1704759020373,
          "duration" : 6181
        }
      }, {
        "uid" : "47a1e1015056825f",
        "status" : "skipped",
        "statusDetails" : "XFAIL reason: Execution blocked by Cloudflare\n\n_pytest.outcomes.XFailed: Execution blocked by Cloudflare",
        "time" : {
          "start" : 1704756556606,
          "stop" : 1704756559235,
          "duration" : 2629
        }
      } ]
    },
    "tags" : [ "smoke" ]
  },
  "source" : "a0994c9c10b396b7.json",
  "parameterValues" : [ ]
}